tile_generator/templates/jobs/deploy-all.sh.erb

```diff
@@ -55,17 +55,22 @@ function import_opsmgr_variables() {
 	{% endif %}
 	{% endfor %}
 	{% endfor %}
+	<% empty_dict = {} %>
+	<% empty_list = [] %>
 	{% for release in context.releases.values() if release.consumes %}
 	{% for link in release.consumes %}
 	<% if_link('{{ link }}') do |link| %>
 	<% hosts = link.instances.map { |instance| instance.address } %>
-	export {{ link | upper }}_HOST="<%= link.instances[0].address %>"
+	export {{ link | upper }}_HOST="<%= link.instances.empty? ? ( puts "" ) : ( puts link.instances[0].address ) %>"
 	export {{ link | upper }}_HOSTS=<%= Shellwords.escape(hosts.to_json) %>
-    export {{ link | upper }}_PROPERTIES=<%= Shellwords.escape(link.properties.to_json) %>
+        export {{ link | upper }}_PROPERTIES=<%= Shellwords.escape(link.properties.to_json) %>
```

Comment:
This line and 67 don't match the indentation of the `do`

Reply:
I noticed the indentation did not seem effective in the output of the job template, so this is just a me correcting the template output to look pretty for reading.